docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -158,6 +158,19 @@ You can add the `greeting.suffix`, remove the other properties, change the value
 As usual, the application can be packaged using `./mvnw clean package` and executed using the `-runner.jar` file.
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+== Programmatically access the configuration
+
+You can access the configuration programmatically,
+this can be handy when needed to do it dynamically, or in a class that is not a CDI bean so `@ConfigProperty` cannot be used.
+
+You need to use the MP's `org.eclipse.microprofile.config.ConfigProvider` to access the configuration of your application.
```

Comment:
Might be better:

You can access the configuration programmatically using the `org.eclipse.microprofile.config.ConfigProvider.getConfig()` such as in:

Reply:
done